packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -121,6 +121,7 @@ const darkTheme = 'darkTheme';
 const denseMode = 'denseMode';
 const analytics = 'analytics';
 const vmDeveloperMode = 'vmDeveloperMode';
+const hoverEvalMode = 'hoverEvalMode';
```

Comment:
nit: `inspectorHoverEvalMode` to make it clear that this setting only corresponds to the inspector (we have hover evals in the debugger too)